docs/meson.build

```diff
@@ -0,0 +1,46 @@
+markdown_bin = find_program(
+  'cmark-gfm', 'redcarpet', 'markdown2', 'markdown', 'markdown_py',
+  disabler: true,
+  required: get_option('docs')
+)
+
+if markdown_bin.found() and markdown_bin.full_path().endswith('cmark-gfm')
+  markdown_args = [ '-e', 'table' ]
+else
+  if markdown_bin.found()
+    warning('The preferred markdown tool cmark-gfm was not found, you will not get Github-style tables.')
+  endif
+  markdown_args = []
+endif
+
+index_md = custom_target(
+  input: [ 'index.py', '../README.md' ],
+  output: 'index.md',
+  command: [ python, '@INPUT@' ],
+  capture: true,
```

Comment:
If you are going to write this script anyway, you might consider making it take an output file too.

Using `capture: true` is a workaround for compile/build programs that don't support output files, and using it incurs the cost of a python wrapper script that reads the stdout of the command and saves it to a file.

The python script seems like it should be part of the command array, not specified as an input file.

Reply:
Output file probably make sense.

If the python script is part of the command array, will it be accounted for as a dependency? (i.e., will changes to the python script result in re-running the build rule?)

Reply:
Yes.

You'll need to refer to the file by its path as `files('bees-usage.py')` to ensure it resolves as a path rather than a string.